commontype/fee_config.go

```diff
@@ -144,6 +145,57 @@ func (f *FeeConfig) checkByteLens() error {
 	return nil
 }
 
+func (c *FeeConfig) getBigIntToSerialize() []**big.Int {
+	return []**big.Int{
+		&c.GasLimit, &c.MinBaseFee, &c.TargetGas, &c.BaseFeeChangeDenominator,
+		&c.MinBlockGasCost, &c.MaxBlockGasCost, &c.BlockGasCostStep,
+	}
+}
```

Comment:
what is this used for? this look a bit weird.

Reply:
@aaronbuchwald Is this kinda of code not desired?  It is a canonical list of numbers and how they are marshal and unmarshal. It is way error prone and it takes advantage of the pointer to a pointer.

Reply:
I see this is trying to make the code simpler by passing a list of pointers to pointers so we can serialize and de-serialize with a for loop instead of one by one.

It's not valid for any of these to be nil, so would it be possible to remove the double pointer and enforce that all of them are non-nil instead of allowing a boolean to indicate whether or not it's nil?

Reply:
Fixed in 2de90955940a692448c1b3fca111b0a93b3db238